ui-tests/src/test/java/org/openmrs/reference/UsingBackButtonInMergePatientTest.java

```diff
@@ -30,23 +30,20 @@
 
     @Before
     public void setUp() throws Exception {
-       
         homePage = new HomePage(page);
-        assertPage(homePage);
+        assertPage(homePage.waitForPage());
```

Comment:
During the test setUp(), many pages are invoked which takes some time for the patient registration page to be loaded by the classLoader. Therefore adding waitForPage() caters for the time taken to load the resource such that the following operations carry on smoothly.

Reply:
> Therefore adding waitForPage() caters for the time taken to load the resource such that the following operations carry on smoothly.

Ideally is this true for the change introduced at code `LN-34` ?

Reply:
Kinda, because without waiting for the homePage to load, the assertion will return null. So yeah the waitForPage(), works to await the loading of the so that the assertPage returns true. @kdaud .